File: javascript/3783.Mirror-Distance-of-an-Integer.js
/**
 * @param {number} n
 * @return {number}
 */
var mirrorDistance = function (n) {
  let nCopy = n;
  let nRev = 0;

  while (nCopy) {
    nRev = nRev * 10 + (nCopy % 10);
    nCopy = Math.floor(nCopy / 10);
  }

  return Math.abs(nRev - n);
};

File: python/3783.Mirror-Distance-of-an-Integer.py
class Solution:
    def mirrorDistance(self, n: int) -> int:
        rev_num = 0
        n_copy = n

        while n_copy:
            digit = n_copy % 10
            rev_num = rev_num * 10 + digit
            n_copy //= 10 

        return abs(n - rev_num)